app/controllers/banneduser_experiment_controller.py

```diff
@@ -93,8 +95,12 @@ def update_experiment(self):
     def _find_eligible_newcomers(self, modactions):
         """Filter a list of mod actions to find newcomers to the experiment.
         Starting with a list of arbitrary mod actions, select mod actions that:
+        - are not for users already in the study,
         - are temporary bans, and
-        - are not for users already in the study.
+        - do not appear to be bots.
+
+        NOTE: We also exclude very new accounts.
+        That happens in `_assign_randomized_conditions` because it requires extra user data.
```

Comment:
This does bug me a bit, to be honest, because it's moving the eligibility logic outside of this method

Reply:
what if we renamed `_find_eligible_newcomers` to `_find_eligible_modactions` for clarity?

Reply:
(the idea being that this method filters modactions, but doesn't deal with `ExperimentThing` instances)

Reply:
Ooh. I see what you mean and there's a clarity to that structure, but I tentatively disagree, because conceptually I think we're tracking banned users, not bans. It seems that this renaming require `_assign_randomized_conditions` to (conceptually) hold the role of something like `_find_eligible_newcomers_and_assign_randomized_conditions`?